cmd/pyroscope/help.txt.tmpl

```diff
@@ -35,18 +35,8 @@ Usage of ./pyroscope:
     	Etcd password.
   -distributor.ring.etcd.username string
     	Etcd username.
-  -distributor.ring.heartbeat-period duration
-    	Period at which to heartbeat to the ring. 0 = disabled. (default 5s)
-  -distributor.ring.heartbeat-timeout duration
-    	The heartbeat timeout after which distributors are considered unhealthy within the ring. 0 = never (timeout disabled). (default 1m0s)
-  -distributor.ring.instance-addr string
-    	IP address to advertise in the ring.
-  -distributor.ring.instance-id string
-    	Instance ID to register in the ring. (default "<hostname>")
   -distributor.ring.instance-interface-names string
-    	Name of network interface to read address from. (default [<private network interfaces>])
-  -distributor.ring.instance-port int
-    	Port to advertise in the ring (defaults to server.http-listen-port). (default 4040)
```

Comment:
Just to double check: we're hiding these config options from the sort output, not removing them, right?

Reply:
Correct those a either remove from the normal output (all has it) or it's re-added with common description.